witness/golang/internal/witness/witness.go

```diff
@@ -66,16 +66,11 @@ type Witness struct {
 // New creates a new witness, which initially has no logs to follow.
 func New(wo Opts) (*Witness, error) {
 	// Create the chkpts table if needed.
-	_, err := wo.DB.Exec(`CREATE TABLE IF NOT EXISTS chkpts (
-			      logID BLOB PRIMARY KEY,
-			      chkpt BLOB,
-			      range BLOB
-			      )`)
-	if err != nil {
-		return nil, err
+	if err := wo.Persistence.Init(); err != nil {
+		return nil, fmt.Errorf("Init(): %v", err)
```

Comment:
"Persistence.Init():" for debug friendliness?

Reply:
Sure.